metrics.coffee

```diff
@@ -60,8 +61,8 @@ module.exports = Metrics =
 
 	injectMetricsRoute: (app) ->
 		app.get('/metrics', (req, res) -> 
-			res.set('Content-Type', Register.contentType)
-			res.end(Register.metrics())
+			res.set('Content-Type', register.contentType)
+			res.end(register.metrics())
```

Comment:
This will break because register is no long a public variable, it lives on `Metrics.register` If you revert line 2 then change line 18 to `register:Register` it will work.

Reply:
It seems to work as it is?

Reply:
Are you sure this is the code that you are testing? i.e. the process has been stopped and started again. Unless I am missreading this then it shouldn't work.

Reply:
```
docker-compose stop
docker system prune
docker system prune
docker network create sharelatex_overleaf
bin/up analytics
```

Reply:
I can change it to your suggestion if you prefer?

Reply:
if you are testing this on analytics then I think `injectMetricsRoute` is never being called? try it on a different service.

Reply:
Good catch, I see what you mean.